frontend/lib/src/theme/globalStyles.ts

```diff
@@ -313,6 +313,12 @@ export const globalStyles = (theme: EmotionTheme): SerializedStyles => css`
     font-weight: ${theme.fontWeights.bold};
   }
 
+  // Override h1 font weight to default weight
+  h1 b,
+  h1 strong {
+    font-weight: ${theme.fontWeights.extrabold};
+  }
```

Comment:
**question:** Is this intentionally modifying a global CSS selector? I'm asking because the PR description and title both refer to "bold in Markdown," but this will apply even outside of the Markdown component such as `st.html("<h1>Hello <strong>world</strong></h1>")`. That's great if that's expected, I just wanted to check!

Reply:
Good callout! This is intended be applied to all headers (e.g. including those set by [`st.title`](https://docs.streamlit.io/develop/api-reference/text/st.title)) per conversation with Jessi. I'll update the PR description